meilisearch-core/src/raw_indexer.rs

```diff
@@ -1,10 +1,11 @@
 use std::borrow::Cow;
 use std::collections::{BTreeMap, HashMap};
 use std::convert::TryFrom;
+use std::println;
```

Comment:
Wut?

Reply:
Lol, rust-analyzer new version handles imports for you, I guess it went a bit out of its way 😅